### DISCUSSION TITLE
Open Bunch Quicksilver action

### DISCUSSION THREAD

**Discussion by anatomatic** [ttscoff/bunch-docs#249] [Show and tell]
Hi all — I made a quick-and-dirty [AppleScript Quicksilver Action](https://gist.github.com/anatomatic/32afbd813ce7e7fc4465fe7c11927a2b)

Finding documentation for Quicksilver Action development is a bit of an archeological exercise (though the project is well-maintained), so I couldn't figure out how to make this action the default action when a .bunch file is in first panel — but still helpful for launching bunches via trusty old [Quicksilver](https://qsapp.com).

**Comment by ttscoff** [comment]
This is great, thanks for sharing!
